test/unit/components/VCard/__snapshots__/VCardMedia.spec.js.snap

```diff
@@ -58,3 +58,16 @@ exports[`VCardMedia.js should render component with custom height (string) and m
 </div>
 
 `;
+
+exports[`VCardMedia.js should render component with images with urls that contain the ( & ) characters 1`] = `
+
+<div class="card__media"
+     style="height: auto;"
+>
+  <div class="card__media__background">
```

Comment:
This test doesn't work, there's no inline style in the snapshot. Then again none of the others do either because jsdom is broken af.

Reply:
Exactly.. I thought of that and I had expected it would somehow inject the style but I looked at the other snaps. So no idea on how to fix this. Itchy though!